src/lib/isThemedPath.ts

```diff
@@ -0,0 +1,7 @@
+export default function isThemedPath(currentPath: string): boolean {
+	return (
+		__config.dev_dot.non_themed_paths.findIndex(
+			(nonThemedPath: string) => nonThemedPath === currentPath
+		) === -1
```

Comment:
⛏️ Very small nitpick: we could use [Array.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) here if we want, might be a little cleaner:

```suggestion
		__config.dev_dot.non_themed_paths.includes(currentPath)
```